bliss/models/flux_net.py

```diff
@@ -0,0 +1,247 @@
+import numpy as np
+
+import pytorch_lightning as pl
+import torch
+from torch import nn
+
+import torch.nn.functional as F
+from torch.distributions import normal
+
+
+from bliss.models import decoder
+from bliss.models.encoder import get_star_bool, tile_images
+from bliss.optimizer import get_optimizer
+
+
+def _trim_images(images, trim_slen):
```

Comment:
Since this is a simple function I think you can remove extra spaces between lines inside this function